pages/proof-of-yield/[timestamp]/[strategy].tsx

```diff
@@ -0,0 +1,539 @@
+import { Link } from "../../../types";
+import { useRouter } from "next/router";
+import Head from "next/head";
+import {
+  Button,
+  Footer,
+  GradientButton,
+  Header,
+  Section,
+} from "../../../components";
+import React, { useMemo, useState } from "react";
+import moment, { Moment } from "moment/moment";
+import Error from "../../404";
+import { GetStaticPaths, GetStaticProps, GetStaticPropsContext } from "next";
+import { assetRootPath, fetchAPI, transformLinks } from "../../../utils";
+import { Container } from "../../../components/Container";
+import {
+  strategies,
+  StrategyInfo,
+} from "../../../sections/proof-of-yield/utils/strategies";
+import { ContainerHeader } from "../../../components/ContainerHeader";
+import { ContainerBody } from "../../../components/ContainerBody";
+import { ExternalLinkButton } from "../../../components/ExternalLinkButton";
+import LineBarChart from "../../../components/LineBarChart";
+import { fetchDailyYields } from "../../../queries/fetchDailyYields";
+import { formatEther } from "viem";
+import Image from "next/image";
+import { useQuery } from "react-query";
+import { twMerge } from "tailwind-merge";
+import { TimeUnit } from "chart.js";
+import Tooltip from "../../../components/proof-of-yield/Tooltip";
+import * as dn from "dnum";
+import { useElementSize } from "usehooks-ts";
+
+const overrideCss = "px-8 md:px-10 lg:px-10 xl:px-[8.375rem]";
+
+const sma = (days: number) => {
```

Comment:
nit: the `periods` closure seems a bit strange to me, but I may be missing the goal

Reply:
An SMA needs to know a history for the length of periods it cares about. So I call the function saying I'm going to track a ~14 period SMA. The returned function is then used to calc the SMA for one specific range of inputs.

```typescript
const sma10 = sma(2)

sma10(5) // 5
sma10(1) // 3
sma10(2) // 1.5
```